### PR TITLE
Update scripts to accommodate operate-first, move common code

### PR DESCRIPTION
**Pull request by khansaad**
This PR contains following changes:

- Updated hpo-demo script to use the app already running on operate-first
- Moved common code in both autotune and HPO demo in a common-helper fie

Signed-off-by: Saad Khan <saakhan@redhat.com>

### CODE REVIEW

File: hpo_demo_setup.sh [hunk 1]
```diff
@@ -15,6 +15,10 @@
 # limitations under the License.
```
Comment (dinogun) [inline hpo_demo_setup.sh:15]: Can you please update this file to have tabs uniformly, currently it is bad mix of both

**Comment by dinogun** [comment]
@khansaad Pls GPG sign all your commits. Looks like some commits are signed and some are not

File: hpo_demo_setup.sh (hunk 1 above)
Comment (khansaad) [reply hpo_demo_setup.sh:15]: done

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**